Lib/random.py

```diff
@@ -261,6 +261,8 @@ def randint(self, a, b):
     def _randbelow_with_getrandbits(self, n):
         "Return a random int in the range [0,n).  Raises ValueError if n==0."
 
+        if n == 0:
+            raise ValueError("Boundary cannot be zero")
```

Comment:
Maybe use `assert`?

Reply:
This is the same error as in the other implementation (the one without `getrandbits`).

Reply:
Seems this check causes a performance regression.

Reply:
Is there any situation where a user could encounter the above `ValueError` from the public methods? If not, I think an assertion would make more sense here; seeing as it's a private member that's only being used from within the `random` module.

Also, how significant is the performance regression @serhiy-storchaka?